.github/workflows/build-deb-pkg.sh

```diff
@@ -0,0 +1,32 @@
+PKG_ROOT=/tmp/avalanche
+DEBIAN_BASE_DIR=$PKG_ROOT/debian
+AVALANCHE_BUILD_BIN_DIR=$DEBIAN_BASE_DIR/usr/local/bin
+AVALANCHE_LIB_DIR=$DEBIAN_BASE_DIR/usr/local/lib/avalanche
```

Comment:
For the Ansible roles I'm currently debating whether the directories should remain `/usr/local/lib/avalanche/...`, or change to `/usr/local/lib/avalanchego/...`. What do you think?

Independent of whether `'plugins` is part of the path.

Reply:
Given that `~/.gecko/` has also been renamed to `~/.avalanchego`, I'm leaning toward using `/usr/local/lib/avalanchego/...`

Reply:
Internally we chose `/usr/local/lib/avalanchego/`